packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/utils/connect.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files